polygeist/tools/cgeist/Test/Verification/sycl/kernels_funcs.cpp

```diff
@@ -0,0 +1,95 @@
+// Copyright (C) Intel
+
+//===--- kernels_funcs.cpp ------------------------------------------------===//
+//
+// Part of the LLVM Project, under the Apache License v2.0 with LLVM Exceptions.
+// See https://llvm.org/LICENSE.txt for license information.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+//
+//===----------------------------------------------------------------------===//
+
+// RUN: sycl-clang.py %s -S | FileCheck %s
+
+#include <sycl/sycl.hpp>
+
+// CHECK-NOT: module
```

Comment:
curious why `gpu.module` doesn't match `module` here.

Reply:
As per [the documentation](https://llvm.org/docs/CommandGuide/FileCheck.html#the-check-not-directive):
> The “CHECK-NOT:” directive is used to verify that a string doesn’t occur between two matches (**or before the first match**, or after the last match).